### ISSUE TITLE
AppLovinSettings: base instance from Resource.Load or Assetdatabase.FindAsset

### ISSUE THREAD

**Issue by TheoSabattie** [AppLovin/AppLovin-MAX-Unity-Plugin#202]
Hello,  

Thanks for your plugin!  
For an organization purpose, we moved MaxSDK folder in another location and we wish to put the configuration outside the MaxSDK folder.  
(Our common ressources is in a git submodule. It's share between projects so we need the configuration outside our common resources folder)

I saw AppLovinSettings in a Resources folder so I hopped moving it in another Resources folder. But it didn't work!

I patched quickly for me: 

```cs    
    public static AppLovinSettings Instance
    {
        get
        {
            if (instance == null)
            {
                instance = Resources.Load<AppLovinSettings>(nameof(AppLovinSettings));

                if (instance)
                    return instance;
//... your instructions
```

But I think it will be better if you fix in your side.  

I'm not sure that you use AppLovinSettings at runtime (?) The script is in an Editor space.  
To prevent confusion, maybe you should put it at another place.  
You can grab it with Assetdatabase.
```cs
AssetDatabase.FindAssets($"t:{nameof(AppLovinSettings)}");
```
Return an array of guids, length to 0 if nothing is found. If something is found => AssetDatabase.GUIDToAssetPath then AssetDatabase.LoadAssetAtPath

Thank you in advance ;) 👍

**Comment by santoshbagadi** [comment]
Hi @TheoSabattie , thank you for reporting this. We also use the resource from other adapters, so, we would have to make sure we keep backwards compatibility there. We'll look into making a fix for this soon and update here when a fix is released.

**Comment by JonathanLiuApp** [comment]
We've just released plugin [version 8.0.0](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/releases/tag/release_8_0_0) which includes UPM support. The new version should fix the issue you are facing.
